compress.go

```diff
@@ -102,6 +102,11 @@ func CompressHandlerLevel(h http.Handler, level int) http.Handler {
 			return
 		}
 
+		if r.Header.Get("Upgrade") != "" {
```

Comment:
See the review comment.

Reply:
Which comment?

Reply:
https://github.com/gorilla/handlers/pull/187#discussion_r466991374v - maybe I misunderstood here.

Reply:
Ah! The proposal I made was already pushed, these 4 lines checking the `Upgrade` header should work fine (I've been using a wrapper implementing exactly this in production on framer.com for many months and haven't run into any issues with WebSockets.)